pulpcore/pulpcore/app/models/repository.py

```diff
@@ -18,6 +18,7 @@ class Repository(Model):
 
         name (models.TextField): The repository name.
         description (models.TextField): An optional description.
+        last_version (models.PositiveIntegerField): A record of the last created version number
```

Comment:
I think it's worth adding a note that the corresponding version may no longer exist, and that's why the max number is tracked here.